clickhouse_plantuml/table.py

```diff
@@ -63,6 +63,15 @@ def __init__(
         self.sampling_key = sampling_key
         self.columns = []  # type: List[Column]
 
+    def parse_additional_dependencies(self):
+        """
+        Add target table of MaterializedView to dependencies
```

Comment:
Move the comment from https://github.com/Felixoid/clickhouse-plantuml/blob/774cfafee30020bf28eeb8ab8c07d6182e32ec22/clickhouse_plantuml/tables.py#L142-L148 here, please